src/sdk/pynni/nni/nas/pytorch/mutables.py

```diff
@@ -56,8 +56,8 @@ def _check_built(self):
                 "Mutator not set for {}. Did you initialize a mutable on the fly in forward pass? Move to __init__"
                 "so that trainer can locate all your mutables. See NNI docs for more details.".format(self))
 
-    def __repr__(self):
-        return "{} ({})".format(self.name, self.key)
+    # def __repr__(self):
+    #     return "{} ({})".format(self.name, self.key)
```

Comment:
Can you do a better repr instead of delete it?

Reply:
the default __repr__ is good enough, it's easy to compare with original network structure.
If we want some additional information, it can be another side method.

Reply:
Then delete it instead of commenting.

Reply:
removed